src/build/tenscript/tenscript.pest

```diff
@@ -0,0 +1,151 @@
+// Created by intellij-pest on 2023-01-25
+// tenscript
+// Author: mitch
+//
+
+fabric_plan = {
+    "(" ~
+    "fabric" ~
+    surface? ~
+    build? ~
+    shape? ~
+    ")"
+}
+
+shape = {
+    "(" ~
+    "shape" ~
+    (space_statement | join_statement | finally_statement)+ ~
+    ")"
+}
+
+space_statement = {
+    "(" ~
+    "space" ~
+    atom ~
+    number ~
+    ")"
+}
+
+join_statement = {
+    "(" ~
+    "join" ~
+    atom ~
+    ")"
+}
+
+finally_statement = {
+    "(" ~
+    "finally" ~
+    atom ~
+    ")"
+}
+
+build = {
+    "(" ~
+    "build" ~
+    seed? ~
+    node? ~
+    ")"
+}
+
+node = {
+    face | grow | mark | branch
+}
+
+face = {
+    "(" ~
+    "face" ~
+    face_name ~
+    node ~
+    ")"
+}
+
+face_name = {
+    ":A-" |":B-" |
+    ":C-" |":D-" |
+    ":A+" |":B+" |
+    ":C+" |":D+"
```

Comment:
These should omit the ':' prefix, no? In our language, identifiers that start with a capital are automatically atoms.

Reply:
Ok